### ISSUE TITLE
Doesn't run on Fedora 32

### ISSUE THREAD

**Issue by tobimensch** [jmechnich/sysstat-applet#3]
I installed all dependencies including SIP with dnf, and then it didn't work with this error message:
```
  File "/usr/local/lib/python3.8/site-packages/appletlib/posixsignal.py", line 1, in <module>
    import signal, sip, socket
ModuleNotFoundError: No module named 'sip'
```

Then I installed more sip related packages with pip3. The error stays the same.

I'm curious why you don't seem to use the official python3 bindings of Qt5? Which are generated with Qt's own shiboken.

It really doesn't matter as long as the stuff works though.

**Comment by jmechnich** [comment]
Hm, that is odd. On Debian I just have to apt install the python3-sip package. If you know of a smoother way of implementing this and possibly get rid of this dependency, I would be happy to merge a PR. It might be that this workaround is not needed anymore with newer versions of Python and PyQt.

What happens if you just ``import sip`` in a ``python3`` interpreter session ?

**Comment by jmechnich** [comment]
I just noticed this in ``pip search sip`` on my machine:

```
sip (5.3.0)                      - A Python bindings generator for C/C++ libraries
  INSTALLED: 4.19.23
  LATEST:    5.3.0
```

If I am not mistaken it would be ``sip5`` for the newer versions.

**Comment by tobimensch** [comment]
So does that mean you're still using Qt4?

```
>>> import sip
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'sip'
```

```
pip install sip
WARNING: Running pip install with root privileges is generally not a good idea. Try `pip install --user` instead.
Requirement already satisfied: sip in /usr/local/lib64/python3.8/site-packages (5.3.0)
Requirement already satisfied: toml in /usr/local/lib/python3.8/site-packages (from sip) (0.10.1)
Requirement already satisfied: packaging in /usr/local/lib/python3.8/site-packages (from sip) (20.4)
Requirement already satisfied: six in /usr/lib/python3.8/site-packages (from packaging->sip) (1.14.0)
Requirement already satisfied: pyparsing>=2.0.2 in /usr/lib/python3.8/site-packages (from packaging->sip) (2.4.7)
```

**Comment by jmechnich** [comment]
> So does that mean you're still using Qt4?
> 

No, I was using sip for deallocating Qt objects (immediately) at certain points in the code. However, I removed the sip dependency now as this function seems to be gone with sip5 anyways, and QObject::deleteLater() is used instead.

You will have to update appletlib to make it work (and manually uninstall any old files first as I have updated the package management as well).
